addons/sale_project/models/sale_order.py

```diff
@@ -273,6 +273,12 @@ def write(self, values):
             self.env['project.project'].sudo().search([('sale_line_id.order_id', 'in', self.ids)]).sale_line_id = False
         return res
 
+    def copy_data(self, default=None):
+        return super().copy_data({
+            'project_id': False,
+            **(default or {}),
+        })
```

Comment:
Why not just set `copy=False` on `project_id` field? 🧐

Reply:
bc I had totally forgotten it was a possibility :see_no_evil: